entrypoint.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-echo "PACKER_VERSION=1.6.6" >> "$GITHUB_ENV"
-echo "TERRAFORM_VERSION=0.12.31" >> "$GITHUB_ENV"
+{ echo "PACKER_VERSION=1.6.6"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=0.14.1"; } >> "$GITHUB_ENV"
```

Comment:
The [versioning for `terraform-docs`](https://github.com/terraform-docs/terraform-docs/releases) has a leading `v`.

```suggestion
{ echo "PACKER_VERSION=1.6.6"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=v0.14.1"; } >> "$GITHUB_ENV"
```

Reply:
Great catch!